services/net/capture/CaptureAction.cs

```diff
@@ -77,6 +90,13 @@ public CaptureAction(IApiService api, IOptions<CaptureOptions> options, ILogger<
 
                 if (reference != null)
                 {
+                    if (manager.DataSource.ContentTypeId > 0)
+                    {
+                        var messageResult = await SendMessageAsync(process, manager.DataSource, schedule, reference);
```

Comment:
If the data source is configured to generate content, then send message to Kafka.